### ISSUE TITLE
Accessibility Issue, Voice Control does not appear to see post or comment content in the Reader.

### ISSUE THREAD

**Issue by zachspears** [wordpress-mobile/WordPress-iOS#17541]
### Expected behavior
Voice Control should be able to see the content of a post excerpt or comment in order to read it to users with accessibility settings enabled. 

### Actual behavior
Voice Control does not see this content at all. 

![IMG_0291](https://user-images.githubusercontent.com/10124024/143093936-353f78bb-29b1-438f-a1fb-daf8cfad4fa3.PNG)
]

### Steps to reproduce the behavior
1. Enable Voice Control with Overlay > Item Names enabled. 
2. Open the Reader to see missing post excerpt content.
3. Select the comments section of a post with at least one comment, you will see the content of the comment is not picked up.

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
